tests/integration/archiveTechRecordStatus.intTest.ts

```diff
@@ -66,9 +66,47 @@ describe("archiveTechRecordStatus", () => {
             expect(techRecordWrapper.techRecord[0].statusCode).toBe(
               STATUS.ARCHIVED
             );
-            expect(techRecordWrapper.techRecord[0].notes).toBe(
-              "string\nTest"
+            expect(techRecordWrapper.techRecord[0].notes).toBe("string\nTest");
+          });
+      });
+    }
+  );
+
+  context(
```

Comment:
With this being an API, could we have a test case for an unhappy path where you attempt to archive a tech record that is already archived but has had it's vin changed?

Reply:
Added additional tests for archiving an archived record where there is multiple and one.